### ISSUE TITLE
[Feature] Obfuscate strings with number.toString

### ISSUE THREAD

**Issue by da411d** [javascript-obfuscator/javascript-obfuscator#681]
My idea to use `number.toString(base)` to obfuscate parst of strings.

For example, `57005..toString(16)` equals `dead`, `29234652..toString(36)` equals `hello`.

Base (aka `radix`) can be any between 2 and 36, but it can encode not all letters. With `2` we can encode only `0` and `1`, with `16` - `0-9a-f` combinations, `36` - `0-9a-z`.

https://gist.github.com/da411d/51a19300c85bb875ce5f4b93ea209bb0

**Comment by sanex3339** [comment]
Interesting

**Comment by da411d** [comment]
Of course, not every matching part of string should be replaced. It may select randomly only some of them
